tests/unit/test_preferences.py

```diff
@@ -10,12 +11,12 @@
     PluginsSetting,
     ValidationException,
 )
-from tests import SearxTestCase
+from searx.plugins import Plugin
 
 locales_initialize()
 
 
-class PluginStub:  # pylint: disable=missing-class-docstring, too-few-public-methods
+class PluginStub(Plugin):  # pylint: disable=missing-class-docstring, too-few-public-methods
```

Comment:
Inheriting the `Plugin` reduces linting noise, and doesn't break anything. Not sure why `Plugin` wasn't used directly, but this keeps the change simple for now.